.circleci/config.yml

```diff
@@ -89,87 +9,47 @@ jobs:
     working_directory: /home/circleci/project
 
     steps:
-      - attach_workspace:
-          at: /home/circleci/project
-      - run:
-          name: Activate virtualenv
-          command: echo "source $CIRCLE_WORKING_DIRECTORY/girder_env/bin/activate" >> $BASH_ENV
-
-      - run:
-          name: Run CTest
-          command: ctest -VV -L 'girder_python|girder_coverage' -E js_coverage
-          working_directory: girder_build
+      - checkout:
+          path: /home/circleci/project
 
       - run:
           name: Run Tox
           command: tox
-          working_directory: girder/plugins/isic_archive
-
-      - run:
-          name: Upload coverage
```

Comment:
Let's see if we can unpin Coverage from `pytest_girder` and re-enable coverage downstream here.